vformer/attention/spatial.py

```diff
@@ -69,6 +69,23 @@ def __init__(
             self.sr = nn.Conv2d(dim, dim, kernel_size=1, stride=1)
 
     def forward(self, x, H, W):
+        """
+
+        Parameters
+        ----------
+            x: torch.Tensor
```

Comment:
The parameter names listed can start in the same column as the Parameters heading. They don't need to be indented.